cartridges/int_klaviyo_core/cartridge/scripts/klaviyo/services.js

```diff
@@ -32,6 +32,8 @@ var KlaviyoEventService = ServiceRegistry.createService('KlaviyoEventService', {
         svc.addHeader('Content-type', 'application/json');
         svc.addHeader('Accept', 'application/json');
         svc.addHeader('revision', '2023-02-22');
+        // TODO: dynamically pull extension version
+        svc.addHeader('X-Klaviyo-User-Agent', 'sfcc-klaviyo/23.7.0');
```

Comment:
No way to dynamically add this today?

Reply:
None that I could find within a reasonable amount of time. @cykolln any ideas?

Reply:
Lets make a story for updating changelogger and adding it to this repo. Then we can either add a versioned variable to the cartridge to pull in here or update these lines directly w/ changelogger on release. We currently don't have changelogger in this repo because we use SFCC's versioning semantic which doesn't play well.

Reply:
Sure, [here's the user story](https://klaviyo.atlassian.net/browse/ECOS-3640?atlOrigin=eyJpIjoiMGYzNzY4NjdlOTZkNGU1Y2JhZDc1NGQ4MGUxY2U5M2MiLCJwIjoiaiJ9) 👍